product/tests/test_views/test_accept_purchase_request_view.py

```
@@ -1,85 +1,106 @@
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertEqual(
-            self.purchase_request.state, PurchaseRequest.State.ACCEPTED
+            response.data["message"], UserNotFoundException.default_detail
         )
 
-    def test_accept_purchase_request_with_another_user(self):
+    def test_verify_email_view_verification_code_is_not_valid(self):
         # Arrange
-        force_authenticate(self.accept_purchase_request, user=self.buyer_user)
+        email = "test@example.com"
+        user = UserFactory(
```

Ruff (0.8.2)

Local variable `user` is assigned to but never used

Remove assignment to unused variable `user`

(F841)